290. Word Pattern.py

```diff
@@ -0,0 +1,34 @@
+from collections import OrderedDict
+
+class Solution:
+    def wordPattern(self, pattern, str):
+        """
+        :type pattern: str
+        :type str: str
+        :rtype: bool
+        """
+        letters = [] #letters from pattern
+        words = [] #unique words
+        sentence = str.split() #each word
+        od = OrderedDict()
+        
+        for word in sentence:
```

Comment:
In python, if you have a list then you can create a set.

`words = set(sentence)`